deployment/ansible/roles/driver.monit/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Download monit
+  get_url: >
+    url=https://mmonit.com/monit/dist/binary/{{ monit_version }}/monit-{{ monit_version }}-linux-x64.tar.gz
```

Comment:
Ah, I see. Curious about the choice of using `docker top`? Not sure I have a great alternative that wouldn't require application level health checks.